code/modules/mob/mob_defines.dm

```diff
@@ -112,7 +112,7 @@
 
 //List of active diseases
 
-	var/list/viruses = list() // replaces var/datum/disease/virus
+	var/list/viruses = list() // list of all diseases in a mob
```

Comment:
I'm not going to do it in this PR, but this needs to be renamed `diseases` some day; people need to get it out of their head that the disease system is purely for viruses; it stifles development.